intake_thredds/cat.py

```diff
@@ -27,6 +27,11 @@ class ThreddsCatalog(Catalog):
     def __init__(self, url: str, driver: str = 'opendap', **kwargs):
         self.url = url
         self.driver = driver
+        if "decode_times" in kwargs:
+            self.xarray_kwargs = {"xarray_kwargs": {"decode_times": kwargs["decode_times"]}}
+            del kwargs["decode_times"]
+        else:
+            self.xarray_kwargs = {}
```

Comment:
It's my understanding that the `xarray_kwargs` don't belong in the `catalog` object, but in the `source` object. From what I am seeing here, `self.xarray_kwargs` isn't be used anywhere else in the `catalog` object. Should we remove this?

Reply:
didnt expect a review yet. just started this PR in my iPad last night... you are right. only needed for `source`.

Reply:
No worries :)... Ping me when this is ready for a round of review

Reply:
I didn't see that this was a **Draft** PR :)